run.sh

```diff
@@ -73,19 +100,27 @@ done
 # them in the docker-compose.yml is not possible
 echo ${WEBLOG_ENV:-} > $SYSTEMTESTS_LOG_FOLDER/.weblog.env
 
-echo ============ Run tests ===================
+echo ============ Run $SYSTEMTESTS_SCENARIO tests ===================
 echo 🔥 Starting test context.
 
 docker inspect system_tests/weblog > $SYSTEMTESTS_LOG_FOLDER/weblog_image.json
 docker inspect system_tests/agent > $SYSTEMTESTS_LOG_FOLDER/agent_image.json
 
 docker-compose up -d
-docker-compose exec -T weblog sh -c "cat /proc/self/cgroup" > $SYSTEMTESTS_LOG_FOLDER/weblog.cgroup
 
+echo "Containers started in background."
+
+# docker-compose exec -T weblog sh -c "cat /proc/self/cgroup" > $SYSTEMTESTS_LOG_FOLDER/weblog.cgroup
```

Comment:
Uncomment this line, it's necessary. I suppose that it was not working with the ENTRYPOINT stuff. It should work now